### DISCUSSION TITLE
[1448440] 卧龙：苍天陨落[本体]

### DISCUSSION THREAD

**Discussion by aaabbbxijipng** [pjy612/SteamManifestCache#144] [Share]
` appid :1448440`
keys:
```
1448441;ef3b4eaabfb5d6bd17666007dda635b99375509180b267fbba485e2df18b4ed0
1448442;4e508b0bc29665862ed41a6e37c28fe63fc758dca12297e0237f28b2df4b2fd7
```
Code:C3A694AA3BEA6FB5
Email:2386953175@qq.com

**Comment by aaabbbxijipng** [comment]
想问下如果内测的密钥list更新了会发邮箱吗

**Reply by pjy612** [reply]
emmm 没懂

**Reply by pjy612** [reply]
DLC 缺 密钥 2232890
https://steamdb.info/app/1448440/depots/

**Reply by aaabbbxijipng** [reply]
我理解就是这个内部的密钥应该是随着分享者变多，游戏列表也会随之增加的吧，那以后的新增加的内测密钥我该怎么获取呢？
DLC的话我过几天回去用电脑看下哈

**Reply by pjy612** [reply]
现在怎么获取 以后就怎么获取啊。。。
内测的话 SAI会自动获取 如果内测了还拿不到 那基本就没在共享里面
